libraries/wallet/wallet.cpp

```diff
@@ -2435,7 +2486,6 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
         }
 
         market_history::order_book wallet_api::get_order_book( uint32_t limit ) {
-            FC_ASSERT( limit <= 1000 );
```

Comment:
Why?

Reply:
This assert duplicate code on server side (there this parameter is limited with 500).